modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/DealRegistrationForm/steps/Review/Review.tsx

```diff
@@ -24,7 +24,7 @@ const Review = ({
 	onCancel,
 	onPrevious,
 }: PRMFormikPageProps & DealRegistrationStepProps) => {
-	const {isSubmitting, values} = useFormikContext<DealRegistration>();
+	const {isSubmitting, status, values} = useFormikContext<DealRegistration>();
```

Comment:
```suggestion
	const {isSubmitting, status: something, values} = useFormikContext<DealRegistration>();
```

seria interessante renomear essa prop antes de utilizar, nao ta claro pra que serve.